### DISCUSSION TITLE
Changing logger messages

### DISCUSSION THREAD

**Discussion by KingKobii** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#475] [Q&A] [answered]
Hey there, just wondering if there is code I can change to make the logger display different messages? The online and offline messages look really similar without color in discord, so I wanted to change the 'online' to something like 'streaming'. went digging through the py files a bit but couldn't find anything.

Thanks :)

**Comment by Tkd-Alex** [comment, marked as answer]
From settings no, but you can easily clone the repository and edit this line: https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/blob/8ffefe71242adbd9112d7519bd411b953c269662/TwitchChannelPointsMiner/classes/entities/Streamer.py#L140
